CONTRIBUTING.md

```diff
@@ -51,6 +51,11 @@ Reference any related issues, links, papers, etc. here.
 Tests will run automatically via GitHub Actions when you open a pull request or
 push new commits to an existing pull request.
 
+With current github permissions, a PR is merge-ready after a single approval
+from any member of @urbit/runtime. Anyone can theoretically hit the merge
+button, but unless a PR is a very urgent hotfix, the PR author should generally
+be deferred to in order to complete the PR.
```

Comment:
"should generally be deferred to in order to complete the PR" is vaguer than it needs to be. Something like the following(as a replacement for the last sentence)  is clearer in my opinion: "Although anyone can hit the merge button, defer to the PR author to merge the PR unless the PR is an urgent hotfix that needs to be merged ASAP".

Reply:
done